aten/src/ATen/native/miopen/Conv_miopen.cpp

```diff
@@ -230,6 +230,7 @@ struct ConvolutionParams
   int dilation[max_dim];
   int64_t groups;
   bool deterministic;
+  int device_id; //This is needed to distinguish between miopen handles of multiple gpus.
```

Comment:
Which places use it?

Reply:
Line 267, see below.

Reply:
I saw that, but it's just setting this field, I don't see anyone is reading/using it though.

Reply:
Its used in Line 494 in the method findAlgorithm.

Reply:
@lcskrishna I see. Thanks for clarification.